abi.html

```diff
@@ -164,10 +164,15 @@ <h3><a href="#definitions"> 1.1 Definitions </a></h3>
 <dt> <i>empty class</i> </dt>
 <dd>
 A class with no non-static data members,
-no unnamed bit-fields other than zero-width bit-fields,
+no unnamed bit-fields other than zero-width bit-fields and empty data members,
```

Comment:
Shouldn't the "empty data members" be a qualification on "no non-static data members" rather than "no unnamed bit-fields"?

Reply:
Yes, absolutely; that was a mistake introduced by my merge.